common/src/main/java/org/wso2/carbon/testgrid/common/InfrastructureProvider.java

```diff
@@ -57,7 +57,7 @@
      * @param deployment An instance of a Deployment which Infrastructure should be removed.
      * @param infraRepoDir - Location of the cloned repository related to infrastructure.
      * @return boolean status of the operation
-     * @throws TestGridInfrastructureException
+     * @throws TestGridInfrastructureException  - If something goes wrong during the infrastructure destroy process.
```

Comment:
Same as the above comment

Reply:
Fixed with 14598a57750f800c12e6cb5a3828474056e7535c.